wbobeirne/joule-extension
# List & connect to peers

Closes #37

### Description

On the path to make Joule a full-featured interface, this adds the ability to list & add peers.

### Steps to Test

1. Open peers list, confirm it matches your node's peers.
2. Add a peer, confirm it works.
3. Add a peer of invalid information, confirm it errors.

## Screenshots

<img width="186" alt="screen shot 2019-01-07 at 7 20 12 pm" src="https://user-images.githubusercontent.com/649992/50801101-46550880-12b1-11e9-97e4-61607232b4e6.png">


<img width="549" alt="screen shot 2019-01-07 at 7 15 28 pm" src="https://user-images.githubusercontent.com/649992/50801081-3b01dd00-12b1-11e9-8ac1-148cac4cd878.png">
